src/observers/CsvObserverBase.hpp

```diff
@@ -260,11 +260,21 @@ inline std::string HeaderTypeToStr(CSVFormat::HeaderType HType)
 
 inline std::string StrToDateFormat(const std::string& FormatStr)
```

Comment:
Check usage of timeindex as DT

Reply:
Stored as boolean `IsTimeIndexDateFormat` in `CSVFormat` object, then used in `saveToFiles` function in CSVFilesObs.cpp

Reply:
"timeindex" not converted in uppercase since not used in `StrToDateFormat` function and used elsewhere with current case.